lib/util/fs/index.spec.ts

```diff
@@ -204,6 +206,51 @@ describe('util/fs/index', () => {
     });
   });
 
+  describe('readLocalSynmlink', () => {
+    it('reads symlink', async () => {
+      await withDir(
+        async (localDir) => {
+          GlobalConfig.set({
+            localDir: localDir.path,
+          });
+          await writeLocalFile('test/test.txt', '');
+          await fs.symlink(
+            resolve(localDir.path, 'test/test.txt'),
+            resolve(localDir.path, 'test/test')
+          );
```

Comment:
This will fail on windows when user isn't running test as administrator.

@zharinov @JamieMagee  any idea how to solve?

Reply:
Wow, but why such a requirement became necessary?

Reply:
@viceice you need to mark this conversation as resolved before we can merge